### PR TITLE
Remove third-party logos from doc images

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Remove third-party logos from doc images:

+ Remove `select-cloud-provider.png` and `tidb-cloud-network-access-dashboard.png`. These interface images are not a must for users to perform the described steps.
+ Cut out the AWS logo from `aws-vpc-guide-1.jpg`, `aws-vpc-guide-4.png`, `aws-vpc-guide-5.png`.

This may help avoid potential legal risks.

### What is the related PR or file link(s)?

Reference link(s):

### CODE REVIEW

**Comment by TomShawn** [comment]
@lilin90 @sivagao PTAL, thanks~

File: create-tidb-cluster.md [hunk 1]
```diff
@@ -15,9 +15,7 @@ This document describes how to create a TiDB Cloud cluster.
 
 2. Select Cloud Provider.
 
-    On the **TiDB Cluster** page, click the cloud service provider icon on the left of the top navigation bar, and select the cloud provider.
-
-    ![Select Cloud Provider](/media/select-cloud-provider.png)
+    On the **TiDB Cluster** page, click the cloud service provider icon on the left of the top navigation bar, and select the cloud provider. The existing options are Amazon Web Services and Google Cloud.
```
Comment (TomShawn) [inline create-tidb-cluster.md:18]:
```suggestion
    On the **TiDB Cluster** page, click the cloud service provider icon on the left of the top navigation bar, and select the cloud provider. The existing options are Amazon Web Services and Google Cloud. 
```

File: create-tidb-cluster.md [hunk 2]
```diff
@@ -15,9 +15,7 @@ This document describes how to create a TiDB Cloud cluster.
...
```
Comment (TomShawn) [inline create-tidb-cluster.md:18]:
```suggestion
    On the **TiDB Cluster** page, click the cloud service provider icon on the left of the top navigation bar, and select the cloud provider. The existing options are Amazon Web Services and Google Cloud.
```

**Review by lilin90 — COMMENTED** [review]
LGTM

**Review by lilin90 — APPROVED** [review]

**Pull request merged**